lib/instance.js

```diff
@@ -525,9 +525,10 @@ Instance.prototype.save = function(options) {
     , updatedAtAttr = this.Model._timestampAttributes.updatedAt
     , createdAtAttr = this.Model._timestampAttributes.createdAt
     , hook = self.isNewRecord ? 'Create' : 'Update'
-    , wasNewRecord = this.isNewRecord;
+    , wasNewRecord = this.isNewRecord
+    , now = Utils.now(this.sequelize.options.dialect);
 
-  if (updatedAtAttr && options.fields.indexOf(updatedAtAttr) === -1) {
+  if (updatedAtAttr && options.fields.length && options.fields.indexOf(updatedAtAttr) === -1) {
```

Comment:
I added this check not to check if `options.fields` are defined, but to check if we update at least one field, because if we update 0 fields then updatedAt should not be changed.